contracts/src/v0.8/functions/dev/1_0_0/FunctionsCoordinator.sol

```diff
@@ -151,7 +153,7 @@ contract FunctionsCoordinator is OCR2Base, IFunctionsCoordinator, FunctionsBilli
     external
     override
     onlyRouter
-    returns (bytes32 requestId, uint96 estimatedCost, uint256 gasAfterPaymentCalculation, uint256 requestTimeoutSeconds)
+    returns (bytes32 requestId, uint96 estimatedCost, IFunctionsRequest.Commitment memory commitment)
```

Comment:
`requestId` and `estimatedCost` are both within the commitment. Do we need to return them in a tuple or can we just return the commitment?

Reply:
Yeah good point. Tweaking it